app/src/main/kotlin/com/simplemobiletools/dialer/fragments/RecentsFragment.kt

```diff
@@ -69,10 +70,18 @@ class RecentsFragment(context: Context, attributeSet: AttributeSet) : MyViewPage
                         }
                     }
                 }
-
                 allRecentCalls = recents
+
+                //Hide private contacts from recent calls
+                if(SMT_PRIVATE in context.baseConfig.ignoredContactSources) {
```

Comment:
do not forget formatting your code, add a space after "if" and after commenting //. Start comments with lowercase letters